packages/eas-cli/src/build/build.ts

```diff
@@ -140,12 +140,34 @@ function handleBuildRequestError(error: any, platform: Platform): never {
     error?.graphQLErrors?.[0]?.extensions?.errorCode === 'EAS_BUILD_DOWN_FOR_MAINTENANCE'
   ) {
     Log.error(
-      'EAS Build is down for maintenance. Try again later. Check https://status.expo.dev/ for updates.'
+      `EAS Build is down for maintenance. Try again later. Check ${link(
+        'https://status.expo.dev/'
+      )} for updates.`
     );
     throw new Error('Build request failed.');
   } else if (error?.graphQLErrors?.[0]?.extensions?.errorCode === 'EAS_BUILD_FREE_TIER_DISABLED') {
     Log.error(
-      'EAS Build free tier is temporarily disabled. Try again later. Check https://status.expo.dev/ for updates.'
+      `EAS Build free tier is temporarily disabled and we are not accepting any new builds. Try again later. ${learnMore(
+        'https://expo.fyi/eas-build-queues'
+      )}`
+    );
+    throw new Error('Build request failed.');
+  } else if (
+    error?.graphQLErrors?.[0]?.extensions?.errorCode === 'EAS_BUILD_FREE_TIER_DISABLED_IOS'
+  ) {
+    Log.error(
+      `EAS Build free tier is temporarily disabled for iOS and we are not accepting any new builds. Try again later. ${learnMore(
+        'https://expo.fyi/eas-build-queues'
+      )}`
+    );
+    throw new Error('Build request failed.');
+  } else if (
+    error?.graphQLErrors?.[0]?.extensions?.errorCode === 'EAS_BUILD_FREE_TIER_DISABLED_ANDROID'
+  ) {
+    Log.error(
+      `EAS Build free tier is temporarily disabled for Android and we are not accepting any new builds. Try again later. ${learnMore(
```

Comment:
```suggestion
      `Free Android builds are temporarily disabled. ${learnMore(
```

what do you guys think about this? no fluff, just gets right to the point

Reply:
(also slightly more accurate because you can be on an "on demand" plan and see this message)

Reply:
I usually prefer error messages which get straight to the point, so I like this one, although I don't have a strong opinion, and the version proposed by @kbrandwijk is also good for me.

Reply:
@kbrandwijk - your call

Reply:
Given how sensitive users (suddenly) are to nuance when it comes to something that's not working/not available, it'd go with the longer version we discussed above in my comment.